### ISSUE TITLE
zero as exit status when threshold equals zero

### ISSUE THREAD

**Issue by MoogyG** [kucherenko/jscpd#182]
My project contains duplicate code.

Why?

```
"jscpd": {
    "threshold": 0,
  },
```
returns 0
but

```
"jscpd": {
    "threshold": 0.00000001,
  },
```
returns 1

For me, both configurations should return 1.

**Comment by kucherenko** [comment]
thank you for the report, fil fix soon

**Comment by kucherenko** [comment]
fixed in jscpd@2.0.1
